lib/components/Keyboard/KeyboardTracking/KeyboardTrackingView/keyboardTrackingView.api.json

```diff
@@ -0,0 +1,86 @@
+{
+  "name": "KeyboardTrackingView",
+  "category": "keyboard",
+  "description": "A UI component that enables 'keyboard tracking' for this view and it's sub-views.\nWould typically be used when you have a TextField or TextInput inside this view.",
+  "notes": "This view is useful only for iOS.",
```

Comment:
Should this be `note` as well? It used to be `@notes`, right?
BTW, I don't think it's working either way ATM :/

Reply:
Yes. fixed

Reply:
I know. We need to add it to the docs generator script